rust/psibase/src/main.rs

```diff
@@ -268,11 +283,29 @@ fn to_hex(bytes: &[u8]) -> String {
     String::from_utf8(result).unwrap()
 }
 
-fn store_sys(sender: AccountNumber, path: &str, content_type: &str, content: &[u8]) -> Action {
+fn validate_compression(compression_level: u32) -> Result<(), anyhow::Error> {
```

Comment:
clap can do this kind of validation: https://docs.rs/clap/latest/clap/struct.Arg.html#method.value_parser

Reply:
Nice, done